src/pages/ServiceGraph/ServiceGraphRouteHandler.tsx

```diff
@@ -53,7 +53,7 @@ export default class ServiceGraphRouteHandler extends React.Component<
       urlParams.get('edges'),
       ServiceGraphRouteHandler.graphParamsDefaults.edgeLabelMode
     );
-    const urlParamGraphType = urlParams.get('graphType') ? urlParams.get('graphType').toUpperCase() : '';
+    const urlParamGraphType = urlParams.get('graphType') ? urlParams.get('graphType') : '';
```

Comment:
why do you perform this ?: check but not for edge label mode? _edgeLabelMode is the same pattern (uses Enum.forValue, gets the urlParams.get('edges') but it does not perform this same check. I think we should be consistent - either do both the way _edgeLabelMode is done, or both the way _graphType is done.

Reply:
It was required when doing the `.toUpperCase()`, now is no longer required, deleting...

Reply:
Removed and now i'm using directly the urlParams.get('graphType')